api/main.py

```
@@ -131,21 +135,10 @@ async def root_health_check():
 
 if __name__ == "__main__":
     import uvicorn  # pyright: ignore[reportMissingImports]
-    uvicorn.run(
-        "api.main:app",
-        host="0.0.0.0",
-        port=8000,
-        reload=True,
-        log_level="info"
-    )
+
+    uvicorn.run("api.main:app", host="0.0.0.0", port=8000, reload=True, log_level="info")
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)